SuperChamps/interfaces/ISuperChampsToken.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: None
+// Super Champs Foundation 2024
+
+pragma solidity ^0.8.24;
+
+import "./IPermissionsManager.sol";
+
+/// @title Interface which allows contracts to reference lock state and permissions manager from the CHAMP token.
+/// @author Chance Santana-Wees (Coelacanth/Coel.eth)
+interface ISuperChampsToken {
```

Comment:
@figs999 what is the use of this interface as superchampstoken is already minted out and it does not implement this interface in the defination

Reply:
Interfaces do not actually need to be included in a contract for them to conform to them.
This interface was included just to be syntactic sugar that exposes specific methods without requiring the entire contract code to be compiled into future contracts.